distribution/docs/src/main/resources/content/_architectures/url-resource-reader.adoc

```diff
@@ -31,6 +31,23 @@ Configure the URL Resource Reader from the ${admin-console}.
 
 See <<{application-prefix}ddf.catalog.resource.impl.URLResourceReader,URL Resource Reader configurations>> for all possible configurations.
 
+=== Configuring Permissions for the URL Resource Reader
+
+Configuring the URL Resource Reader to retrieve files requires adding Security Manager read permissions to the directory containing the resources. The following permissions, replacing <DIRECTORY_PATH> with the path of the directory containing resources should be placed in the URL Resource Reader section inside ${home_directory}/security/configurations.policy.
+
+.Adding New Permissions
+[WARNING]
+====
+After adding permissions, a system restart is required for them to take effect.
+====
+
+. permission java.io.FilePermission "<DIRECTORY_PATH>", "read";
+. permission java.io.FilePermission "<DIRECTORY_PATH>${/}-", "read";
+
+Trailing slashes after <DIRECTORY_PATH> have no effect on the permissions granted. For example, adding a permission for "/test/path" and "/test/path/" are equivalent. The recursive forms "/test/path${/}-", and "/test/path/${/}-" are also equivalent.
+
+Line 1 gives the URL Resource Reader the permissions to read from the directory. Line 2 gives the URL Resource Reader the permissions to recursively read from the directory, specified by the directory path's suffix "${/}-".
+
```

Comment:
The explanation of what the permission entries do should come before the caveats about trailing slashes. 
Also, numbering the permissions makes it looks like those numbers should be literally added to the file, which I don't think is the case (I could be wrong).  Lastly, I think it would be good to provide the entire context for the grant block. Consider replacing lines 44-49 with:

> grant codeBase "file:/org.apache.tika.core/catalog-core-urlresourcereader" {
>      //# Add permissions here
>      permission java.io.FilePermission "<DIRECTORY_PATH>", "read";
>      permission java.io.FilePermission "<DIRECTORY_PATH>", "read";
> }
> 
> The first permission entry allows the URL Resource Reader to read the directory. The second permission allows the URL Resource Reader to recursively read the contents of the directory. 
>
> The trailing slashes after <DIRECTORY_PATH> have no effect on the permissions granted. For example, adding a permission for "/test/path" and "/test/path/" are equivalent. The recursive forms "/test/path${/}-", and "/test/path/${/}-" are also equivalent.

Reply:
agreed, rearranged this section